beacon-chain/p2p/utils.go

```diff
@@ -93,7 +95,7 @@ func privKey(cfg *Config) (*ecdsa.PrivateKey, error) {
 		return nil, err
 	}
 
-	log.Info("Wrote network key to file")
+	log.WithField("file", defaultKeyPath).Info("Wrote network key to")
```

Comment:
This log looks a little funny since it isn't really a complete sentence. It looks a bit better like this
```suggestion
	log.WithField("filepath", defaultKeyPath).Info("Wrote network key to file.")
```
For comparison:
```
INFO[0000] Wrote network key to                          file=path/to/default/key
```
vs
```
INFO[0000] Wrote network key to file.                    filepath=path/to/default/key
```

Reply:
My initial intent was to avoid having two times the word `file`.

Let me propose an alternative!

Reply:
Fixed in 1d081396ea789ed8aac3ec28c11e487f0bb89052.